src/script/view_model/content/GroupCreationViewModel.ts

```diff
@@ -72,6 +73,7 @@ export class GroupCreationViewModel {
     private readonly userRepository: UserRepository,
   ) {
     this.isTeam = this.teamRepository.isTeam;
+    this.ConversationRepository = ConversationRepository;
```

Comment:
Can't you use the existing `this.conversationRepository`?

Reply:
can't access static method on an instance :(